internal/dag/gatewayapi_processor.go

```diff
@@ -1009,9 +1010,31 @@ func (p *GatewayAPIProcessor) computeHTTPRoute(route *gatewayapi_v1alpha2.HTTPRo
 				if redirect == nil && filter.RequestRedirect != nil {
 					redirect = filter.RequestRedirect
 				}
+			case gatewayapi_v1alpha2.HTTPRouteFilterRequestMirror:
+				// Get the mirror filter if there is one. If there are more than one
+				// mirror filters, "NotImplemented" condition on the Route is set to
+				// status: True, with the "NotImplemented" reason.
+				if mirrorPolicy != nil {
+					routeAccessor.AddCondition(status.ConditionNotImplemented, metav1.ConditionTrue, status.ReasonNotImplemented, "HTTPRoute.Spec.Rules.Filters: Only one mirror filter is supported.")
+					continue
+				}
```

Comment:
Ultimately I think it makes sense to support multiple RequestMirror filters per rule, since the semantics seem pretty clear: if you have >1 RequestMirror filter, then each one defines a backend to mirror traffic to. Envoy easily supports this (see `request_mirror_policies` in https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/route/v3/route_components.proto#config-route-v3-routeaction), and the change to Contour should be pretty straightforward too: change the `dag.Route.MirrorPolicy` to be a slice and then update the places that use it appropriately.

I'm OK if we log an issue and tackle this improvement separately, or if you want to include it in this PR, great!

Reply:
ref. #4566